sdk/open_api/models/depth.py

```diff
@@ -115,12 +115,16 @@ def from_dict(cls, obj: Optional[Dict[str, Any]]) -> Optional[Self]:
         if not isinstance(obj, dict):
             return cls.model_validate(obj)
 
+        # Handle updatedAt which may come as string from API
+        updated_at = obj.get("updatedAt")
+        if isinstance(updated_at, str):
+            updated_at = int(updated_at)
         _obj = cls.model_validate({
             "symbol": obj.get("symbol"),
             "type": obj.get("type"),
             "bids": [Level.from_dict(_item) for _item in obj["bids"]] if obj.get("bids") is not None else None,
             "asks": [Level.from_dict(_item) for _item in obj["asks"]] if obj.get("asks") is not None else None,
-            "updatedAt": obj.get("updatedAt")
+            "updatedAt": updated_at
```

Comment:
@danigr99727 I think the spec is actually a bit wrong, in that we define `updatedAt` to be unsigned integer but we actually are returning string from API ! This confuses AsyncAPI generation, I think we should change `updatedAt` in our spec to actual string, let's discuss !